force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -258,11 +358,10 @@ public with sharing class VariablesDataTypesOperators {
      */
     public static Integer performDivisionAndCast(Double a, Double b) {
         // Perform the division and cast (round down) off the result.
-        Double divisionResult = null;
-
+        Double divisionResult = a / b;
         // Write the code for type casting the divisionResult to an Integer
-        Integer roundedResult = null;
-        return null; // Replace null with the variable you used to store the result
+        Integer roundedResult = divisionResult.intValue();
```

Comment:
Try casting to an integer

Reply:
@walters954 I'm confused on this one.

per Apex Reference for Double Class (https://developer.salesforce.com/docs/atlas.en-us.apexref.meta/apexref/apex_methods_system_double.htm)

`intValue()`
Returns the Integer value of this Double by casting it to an Integer.

Were you wanting me to use `Integer.valueOf(divisionResult)?`